packages/manager/modules/nasha/src/components/metrics/metrics.template.html

```diff
@@ -91,7 +91,7 @@
                     <span>&nbsp;</span>
                     <em
                         class="oui-icon pointer oui-icon-help-circle"
-                        data-responsive-popover="'nasha/information/nasha-information-usage-help.html'"
+                        data-uib-popover="{{:: ('nasha_components_metrics_space_usage_help' | translate) }}"
```

Comment:
I don't see any other place in the manager where data-uib-popover is used. We find oui-popover.

Reply:
Fixed here https://github.com/ovh/manager/pull/6868/files#diff-c57b6aa2acd3255eb6e956279bfa83f8df7a4f9dae91b68c0ffbdc7bed920213